apps/web/pages/auth/sso/[provider].tsx

```diff
@@ -69,7 +70,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
 
   if (session) {
     // Validating if username is Premium, while this is true an email its required for stripe user confirmation
-    if (usernameParam && session.user.email) {
+    if (usernameParam && session.user.email && IS_CALCOM) {
```

Comment:
Since this is now checked in IS_CALCOM and also not calcom envs. we need to only check in calcom envs